tests/ModelGeneratorTest.php

```diff
@@ -98,6 +100,20 @@ className: WarningEvent::class,
         );
     }
 
+    public function testCreateModelByCommand()
+    {
+        $this
+            ->artisan('make:entity Post -S name -t reviewed_at -T publiched_at --only-model')
+            ->assertSuccessful();
+
+        $this->assertGeneratedFileEquals('generated_model.php', 'app/Models/Post.php');
```

Comment:
let's use the same fixture here

```suggestion
        $this->assertGeneratedFileEquals('new_model.php', 'app/Models/Post.php');
```